client/homebrew/pages/archivePage/archivePage.jsx

```diff
@@ -0,0 +1,443 @@
+                <div className="foundBrews searching">
+                    <h3 className="searchAnim">Searching</h3>
+                </div>
+            );
+        }
+
+        if (title === '') {
+            return (
+                <div className="foundBrews noBrews">
+                    <h3>Whenever you want, just start typing...</h3>
+                </div>
+            );
+        }
+
+        if (error) {
+            console.log('render Error: ', error);
+            let errorMessage;
+            switch (error.errorCode) {
+                case '404':
+                    errorMessage = "404 - We didn't find any brew";
+                    break;
+                case '503':
+                    errorMessage =
+                        '503 - Service Unavailable, try again later, sorry.';
+                    break;
+                case '500':
+                    errorMessage =
+                        "500 - We don't know what happened, go ahead and contact the mods or report as a mistake.";
+                    break;
+                default:
+                    errorMessage = 'An unexpected error occurred';
+            }
+
+            return (
+                <div className="foundBrews noBrews">
+                    <h3>Error: {errorMessage}</h3>
+                </div>
+            );
+        }
+
+        if (!brewCollection || brewCollection.length === 0) {
+            return (
+                <div className="foundBrews noBrews">
+                    <h3>No brews found</h3>
+                </div>
+            );
+        }
+
+        return (
+            <div className="foundBrews">
+                <span className="totalBrews">
+                    Brews found: {this.state.totalBrews}
+                </span>
+                {brewCollection.map((brew, index) => (
+                    <BrewItem
+                        brew={brew}
+                        key={index}
+                        reportError={this.props.reportError}
+                    />
+                ))}
+                {this.renderPaginationControls()}
+            </div>
+        );
+    },
+
+    render: function () {
+        return (
+            <div className="archivePage">
+                <link href="/themes/V3/Blank/style.css" rel="stylesheet" />
+                <link href="/themes/V3/5ePHB/style.css" rel="stylesheet" />
+                {this.renderNavItems()}
+
+                <div className="content">
+                    <div className="welcome">
+                        <h1>Welcome to the Archive</h1>
+                    </div>
+                    <div className="flexGroup">
+                        <div className="form dataGroup">
+                            {this.renderForm()}
+                        </div>
+                        <div className="resultsContainer dataGroup">
+                            <div className="title">
+                                <h2>Your searched returned these results</h2>
```

Comment:
This line could report the number of brews found.

- "230 brews matched your query"
- "230 brews returned"
- "Your search returned 230 brews"
- etc

Also note, at minimum, there is a typo, should be "Your search returned these results"

Reply:
I'd rather we don't do that, since the total count takes many seconds to load, there is a counter at the bottom right of the page for this, not disrupting.